doc/qa.rst

```diff
@@ -0,0 +1,92 @@
+.. highlight:: rest
+
+********
```

Comment:
Perhaps we should add a note here to tell the user to check all warnings printed by PYPIT during reduction, to check that their data are being reduced according to their intention.